requirements.txt

```diff
@@ -3,7 +3,7 @@ pytest
 codecov
 pytest-cov
 black
-starlette
+git+git://github.com/encode/starlette@master#egg=starlette
```

Comment:
I've just pushed `0.12.0.b1` - so you can use that if you want.

(Because of the "beta" indicator PyPy marks it as a pre-release, so "pip install starlette" will still get 0.11.4 for now)

Reply:
Will pin to that release for now. Thanks!